### ISSUE TITLE
Bug with link following an emoji

### ISSUE THREAD

**Issue by rileytg** [sourcelair/xterm.js#729]
Writing this content outputs double url (first is clickable):
```js
const ROCKET_EMOJI = "🚀"
const content `${ROCKET_EMOJI}  https://github.com/sourcelair/xterm.js/issues/729`
```

<img width="443" alt="storybook" src="https://user-images.githubusercontent.com/692817/27554387-1b4048ec-5a63-11e7-9cd4-7f13708b37ae.png">

## Details
- Browser and browser version: Chrome 59.0.3071.104
- OS version: macOS 10.12.4 
- xterm.js version: 
```
xterm@^2.7.0:
  version "2.7.0"
  resolved "https://registry.yarnpkg.com/xterm/-/xterm-2.7.0.tgz#05e42d270ff5e67cca95116b62c84c0cfc435baa"
```

### Steps to reproduce

1. write `content` from above to an new xterm instance
2. see mayhem 

(I confirmed both emoji and link work on their own in my setup. its running inside a react app but ive isolated it so that shouldn't be a factor) 

UPDATE:
```js
const content `${ROCKET_EMOJI} \n https://github.com/sourcelair/xterm.js/issues/729`
```
works just vnice

**Comment by Tyriar** [comment]
Duplicate https://github.com/sourcelair/xterm.js/issues/642, fix coming in 2.8.0

**Comment by rileytg** [comment]
womp womp sorry 😞
